### PR TITLE
feat : added homepage with footer and header

### PR DESCRIPTION
**Pull request by aatishpandey**
Added Homepage layout and styles.
Changed Files:
App.js
Footer.jsx
Header.jsx
index.js
home-page.jsx
Routes.js

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,34 +10,15 @@
...
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+    <link rel="stylesheet" href="https://mirageui.netlify.app/components/styles.css">
+    <link rel="stylesheet" href="https://sahajstore.netlify.app/css/styles.css"> -->
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
```
Comment (shivangisharma01) [inline public/index.html:22]: try not give id/class name likes this (as it's used for the root element)

**Review by shivangisharma01 — APPROVED** [review]
just one suggestion other than that looks nice.

File: public/index.html (hunk 1 above)
Comment (aatishpandey) [reply public/index.html:22]: this is the default code in react

**Review by Priyanshi-Tiwari — APPROVED** [review]
the setup and the code looks good

**Review by aishwarya-bm — APPROVED** [review]
Code looks clean and readable. I like the separate component for routes.

**Pull request merged**